### PR TITLE
feat: use new menu at organization level

### PR DESCRIPTION
**Pull request by gcusnieux**
## Issue

https://gravitee.atlassian.net/browse/APIM-1832

## Description

A small description of what you did in that PR.

## Additional context

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-dxbsqxlqqs.chromatic.com)

### CODE REVIEW

**Review by a-cordier — COMMENTED** [review]
can we take advantage of this PR to lock `organization|audit` and `environment|audit` depending on the apim-audit-trail feature ? The table we have describes those two menu entries as locked when the feature is missing

**Comment by gcusnieux** [comment]
> can we take advantage of this PR to lock `organization|audit` and `environment|audit` depending on the apim-audit-trail feature ? The table we have describes those two menu entries as locked when the feature is missing

done ;)

**Comment by a-cordier** [comment]
> > can we take advantage of this PR to lock `organization|audit` and `environment|audit` depending on the apim-audit-trail feature ? The table we have describes those two menu entries as locked when the feature is missing
> 
> done ;)

I think we must add this to the environment menu as well but maybe we can deal with that in another PR

**Comment by ThibaudAV** [comment]
i just add screen capture here
image

File: gravitee-apim-console-webui/src/organization/configuration/navigation/org-navigation.component.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+@use 'sass:map';
+
+.org-navigation {
+  display: flex;
+  flex-direction: row;
+  height: 100vh;
+
+  &__content {
+    overflow: auto;
+    min-height: 100vh;
```
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/organization/configuration/navigation/org-navigation.component.scss:10]: ok no more topbar that why its "work" xD

**Review by ThibaudAV — APPROVED** [review]

File: gravitee-apim-console-webui/src/organization/configuration/navigation/org-navigation.component.ts [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    this.appendConsoleItems();
+    this.appendUserManagementItems();
+    this.appendGatewayItems();
+    this.appendNotificationsItems();
+    this.appendAuditItems();
+    this.appendCockpitItems();
+  }
+
+  navigateTo(route: string) {
+    this.ajsState.go(route);
```
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/organization/configuration/navigation/org-navigation.component.ts:66]: For your information, this is no longer necessary (normally).
now the router and components are angular we can use the ui-sref directive to make links in html

**Pull request merged**